tools/amplify-preview/main.go

```diff
@@ -0,0 +1,136 @@
+/*
+Copyright 2024 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"context"
+	"errors"
+	"log/slog"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/alecthomas/kingpin/v2"
```

Comment:
For consistency, let's use either the flag package or cobra.

E.g.
https://github.com/gravitational/shared-workflows/blob/0f1f9dec1c978a238eae82bfa92a3d6357ad3f1e/bot/main.go#L118-L131
https://github.com/gravitational/cloud/blob/master/tcctl/release/cli/cli.go

Reply:
All tools in https://github.com/gravitational/shared-workflows/tree/master/tools are using `github.com/alecthomas/kingpin` and I followed their example for consistency 😬

Reply:
Not sure why kingpin was chosen for those tools. The review bot uses the simple flag package. We previously used kingpin in cloud but changed to cobra because it has a better design.

Reply:
I think it's because teleport uses kingpin everywhere:
- https://github.com/search?q=repo%3Agravitational%2Fteleport%20kingpin&type=code

Reply:
I would prefer to keep it as is to avoid spending more time on this, as long term we would prefer using native integration, once Amplify get to resolve current issues.